src/inspector/view/qml/MuseScore/Inspector/InspectorForm.qml

```diff
@@ -70,9 +71,20 @@ Rectangle {
     Flickable {
         id: flickableArea
 
-        anchors.fill: parent
+        anchors.top: root.top
+        anchors.right: root.right
+        anchors.left: root.left
         anchors.margins: 12
 
+        interactive: true
+        clip: true
+        flickableDirection: Flickable.VerticalFlick
+        boundsBehavior: Flickable.StopAtBounds
+        maximumFlickVelocity: 1000
+
+        height: root.height
+        contentHeight: contentItem.childrenRect.height + 12 //For space in the bottom
```

Comment:
if you return anchors.fill: parent you will remove space

Reply:
This is the right decision to make contentItem binding. But there is one problem.
In the updateContentHeight function, we will remove this binding when we decide to assign a constant value to the height
`flickableArea.contentHeight = resultContentHeight`
Binding will no longer work. (For testing: collapse the section on the panel, shrink the window to minimum size, expanded the section on the panel, try to scroll)

I think we can remove the updateContentHeight function and save contentItem binding.
But the solution needs to be tested